recipes/python-dp/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "python-dp" %}
+{% set version = "0.1.7dev" %}
+{% set commit = "1b1dc6639173c0a13613189ec21851604a4c7335" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/OpenMined/PyDP/archive/v{{ version }}.tar.gz
+    sha256: 719e5a424bc60862d264ea4e712e9f74699b127b06910be6e6b70b4abd5c4147
+
+  - url: https://github.com/google/differential-privacy/archive/{{ commit }}.tar.gz
+    sha256: 03a3c53be6be5dbd05addd2c27ce997aba8635ee89376f7618d01ee628fb6690
+    fn: differential-privacy.tar.gz
+    folder: third_party/differential-privacy
```

Comment:
Please do not vend other packages and instead depend on them

Reply:
I don't know how should I proceed. Does this mean that I should provide two different recipes? The project provides Python bindings for the Google Differential Privacy C++ library, so we need to fetch the source code to compile our Python extension.

Is there any other recipe that I can use as reference?

Reply:
This means that you should put in a PR here for a differential-privacy package and all for all of the other packages that are not yet on conda-forge

Reply:
The ` differential-privacy package` is a c++ code rather than python code-base. It doesn't make sense to provide c++ code on conda-forge

Reply:
We package a lot of C++ projects. Conda-forge came out of a need to package multiple different languages packages together, as well as projects that have inter-language bindings.  This is all very normal